web/src/parse-resources.tsx

```diff
@@ -0,0 +1,57 @@
+import { Resource } from './logs.types';
+import { notUndefined } from './value-utils';
+
+export enum KindLabel {
```

Comment:
```suggestion
export enum Resource {
```

Reply:
Updated `KindLabel` to `ResourceLabel` since `Resource` is already taken.